src/domain/matomo/Matomo.tsx

```diff
@@ -25,6 +25,7 @@ const matomoInstance = createMatomoInstance({
 function Matomo({ children }: { children: React.ReactNode }): JSX.Element {
   if (matomoInstance) {
     return (
+      // @ts-ignore 2322
```

Comment:
This is not needed if matomo-tracker-react is changed from @datapunt/matomo-tracker-react to @jonkoops/matomo-tracker-react

Reply:
IMHO yes, it's an easy fix, nothing else but `yarn remove @datapunt/matomo-tracker-react` and `yarn add @jonkoops/matomo-tracker-react` and replacing all `@datapunt` with `@jonkoops` in the source code and it should be done. BTW @jonkoops/matomo-tracker-react is the one used in events-helsinki-monorepo.

About the packages one possible problem is that [@jonkoops/matomo-tracker-react](https://github.com/jonkoops/matomo-tracker) v0.7.0 is the last release from 2022-05-23 and the repository is no longer maintained. 

snyk.io gives health score [53/100](https://snyk.io/advisor/npm-package/@datapunt/matomo-tracker-react) for @datapunt/matomo-tracker-react v0.5.1 and [63/100](https://snyk.io/advisor/npm-package/@jonkoops/matomo-tracker-react) for @jonkoops/matomo-tracker-react v0.7.0 so that is slightly in favor of the latter.

@datapunt/matomo-tracker-react v0.5.1 was last published 2021-11-08 according to [npmjs.com](https://www.npmjs.com/package/@datapunt/matomo-tracker-react).

Taking all of these into account I would say changing to @jonkoops/matomo-tracker-react is reasonable even though it is not maintained anymore. I have not looked into any third alternative to these but if there would be that could be an option as well. But as this should be just a direct replacement and fixes the type issue here I think this is a reasonable thing to do.